### PR TITLE
Issues solution sg

### LINKED ISSUE #2: Missing and inproper use of functions
there are no functions

### PR DESCRIPTION
**Pull request by Groneberg**
modified to solve various problems  
#2 #3 #4 #6 #7

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -4,19 +4,62 @@
...
+        break;
+    */
+    default:
+      console.error('Not implemented:', operator);
+  }
+  console.log(result);
+}
+
+// returns the parsing from a string to a number
+function toNumber(x) {
```
Comment (Groneberg) [inline index.js:41]: change to take into account comma numbers

File: index.js [hunk 2]
```diff
@@ -4,19 +4,62 @@
 
 var arguments = process.argv.slice(2);
 
-var operand1 = arguments[0];
-var operand2 = arguments[1];
+var operand1 = toNumber(arguments[0], 10);
+var operand2 = toNumber(arguments[1], 10);
```
Comment (Qubus0) [inline index.js:8]: Maybe use [Number() [MDN]](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number) instead

**Comment by dwdab** [comment]
@Groneberg please consider using more transparent PR title. Thanks!

File: index.js (hunk 1 above)
Comment (dwdab) [reply index.js:41]: 👍

**Pull request merged**